deploy_config_generator/output/kube_servicemonitor.py

```diff
@@ -0,0 +1,334 @@
+import copy
+
+from deploy_config_generator.utils import yaml_dump
+from deploy_config_generator.output import kube_common
+
+
+class OutputPlugin(kube_common.OutputPlugin):
+
+    NAME = 'kube_servicemonitor'
+    DESCR = 'Kubernetes serviceMonitor output plugin'
```

Comment:
While this description is only used for humans, `ServiceMonitor` should be capitalized to match the actual resource name